src/main/webapp/trip/details.js

```diff
@@ -0,0 +1,17 @@
+function select() {
```

Comment:
Is this to select from the dropdown?

Reply:
it is, but so far it only dropdown for the first list. I haven't figured out make them both dropdown